File: C++/Problems/Problems/problem_2651.cpp
#include "problem_2651.h"

bool problem_2651::test() {
    int arrivalTime = 15;
    int delayedTime = 5;

    int expected = 20;

    auto result = findDelayedArrivalTime(arrivalTime, delayedTime);

    std::cout << result << std::endl;

    return result == expected;
}

int problem_2651::findDelayedArrivalTime(int arrivalTime, int delayedTime) {
    return (arrivalTime + delayedTime) % 24;
}

File: Python/Problems/Problems/problem2651.py
import heapq
import math
import sys

from utils import print1DArray, print2DArray
from typing import List, Set, Optional
from collections import Counter

from .problem import Problem

class Problem2651(Problem):
    def test(self) -> bool:
        arrivalTime = 15
        delayedTime = 5

        expected = 20

        result = self.findDelayedArrivalTime(arrivalTime, delayedTime)

        print(result)

        return result == expected

    def findDelayedArrivalTime(self, arrivalTime: int, delayedTime: int) -> int:
        return (arrivalTime + delayedTime) % 24